.gitignore

```diff
@@ -4,5 +4,7 @@ pkg/*
 *.log
 Gemfile.lock
 gemfiles/*.lock
+vendor
+gemfiles/vendor
```

Comment:
put this in your global ignores to never accidentally add it / to avoid having to ignore it in every project